### DISCUSSION TITLE
Fail to open stream require(contrib/yarn.php)

### DISCUSSION THREAD

**Discussion by Edvinaz** [deployphp/deployer#3409] [Help needed] [answered]
Hello, stuck with the problem... I used deployer to deploy to production from command line, but decided to implement github actions to automatically deploy, but i getting error that fails to open stream: 
`PHP Warning: require(contrib/npm.php): Failed to open stream: No such file or directory in /home/runner/work`

used official Github Actions example from Deployer documentation

**Comment by antonmedv** [comment]
Try to run dep locally. Follow install instructions

**Reply by Edvinaz** [reply]
locally it works fine, but github action fails, is there some instructions to follow what could be wrong?

**Comment by Edvinaz** [comment, marked as answer]
found out that i needed to use composer install before deploying
